pom.xml

```diff
@@ -54,6 +54,7 @@
             <plugin>
                 <groupId>org.apache.sling</groupId>
                 <artifactId>maven-sling-plugin</artifactId>
+                <version>2.3.2</version>
```

Comment:
Please inherit the version from the parent

Reply:
Thank you for spotting this. Fixed.